File: QuizApp/src/QuizApp.BLL/DTO/TestResult/TestResultDetailDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

using QuizApp.BLL.Dto.ResultAnswer;

namespace QuizApp.BLL.Dto.TestResult
{
    public class TestResultDetailDto
    {
        public int Id { get; set; }

        public string IntervieweeName { get; set; }

        public DateTime PassingStartTime { get; set; }

        public DateTime PassingEndTime { get; set; }

        public int Score { get; set; }


        public ICollection<ResultAnswerDetailDto> ResultAnswers { get; set; } = new List<ResultAnswerDetailDto>();
    }
}

File: frontend/src/app/models/test-result/test-result-detail-dto.ts
import { ResultAnswerDetailDto } from '../result-answer/result-answer-detail-dto';
import { ResultTestDto } from '../test/result-test-dto';

export interface TestResultDetailDto {
    id: number;
    intervieweeName: string;
    passingStartTime: Date;
    passingEndTime: Date;
    score: number;

    test: ResultTestDto;
}
